cmd/troubleshoot/cli/run_nocrd.go

```diff
@@ -68,17 +74,46 @@ func runTroubleshootNoCRD(v *viper.Viper, arg string) error {
 		return fmt.Errorf("unable to parse %s collectors", arg)
 	}
 
-	archivePath, err := runCollectors(v, collector)
+	s := spin.New()
+	finishedCh := make(chan bool, 1)
+	progressChan := make(chan string, 10)
```

Comment:
What's the reason for 10? Simply to prevent blocking? This should be really quick though, what about making this lower? What's the risk there?

Reply:
no risk